guides/plugins/plugins/framework/message-queue/add-message-to-queue.md

```diff
@@ -10,7 +10,7 @@ nav:
 ## Overview
 
 ::: warning
-Parts of this guide refer to the `async_low_priority` queue and the corresponding `AsyncLowPriorityMessageInterface` which is only available in version 6.5.7.0 and above. Configuring the messenger to consume this queue will fail, if it does not exist.
+Parts of this guide refer to the `low_priority` queue and the corresponding `AsyncLowPriorityMessageInterface` which is only available in version 6.5.7.0 and above. Configuring the messenger to consume this queue will fail, if it does not exist.
```

Comment:
After our change it's also possible to override the transport for specific messages, we should mention this, too.

See the changelog in our internal Gitlab, where it's described

Reply:
I mentioned the possibility to lower the priority via framework.yaml and via the interface. Further more I mentioned overriding transports on a message level.